src/nuclio/functions/function-collapsing-row/function-collapsing-row.component.js

```diff
@@ -260,17 +260,19 @@
         /**
          * Deletes function from functions list
          * @param {Object} [functionItem]
+         * @param {Boolean} [ignoreValidation] - determines whether to forcibly remove the function
          * @returns {Promise}
          */
-        function deleteFunction(functionItem) {
+        function deleteFunction(functionItem, ignoreValidation) {
             if (lodash.isEmpty(apiGateways)) {
                 ctrl.isSplashShowed.value = true;
 
                 return ctrl.handleDeleteFunction({
-                    functionData: lodash.defaultTo(functionItem, ctrl.function).metadata
+                    functionData: lodash.defaultTo(functionItem, ctrl.function).metadata, ignoreValidation: ignoreValidation
```

Comment:
Please start the `ignoreValidation` on a new line here
```suggestion
                    functionData: lodash.defaultTo(functionItem, ctrl.function).metadata,
                    ignoreValidation: ignoreValidation
```